pallene/assignment_conversion.lua

```diff
@@ -343,14 +344,11 @@ function Converter:visit_exp(exp)
         end
 
     elseif tag == "ast.Exp.Var" then
-        local var = exp.var
-        self:visit_var(var, function (new_var)
-            exp.var = new_var
-        end)
-
-    elseif tag == "ast.Exp.Unop" 
-        or tag == "ast.Exp.Cast" 
-        or tag == "ast.Exp.ToFloat" 
+        self:visit_var(exp.var, function (new_var) exp.var = new_var  end)
+
+    elseif tag == "ast.Exp.Unop"
+        or tag == "ast.Exp.Cast"
+        or tag == "ast.Exp.ToFloat"
```

Comment:
The diff is a bit wonky here because of the whitespace changes. It might be easier to read if you look at one commit at a time.

Reply:
It was okay to read, I'm used to strange diffs hehe